e2e/basic-electron.spec.ts

```diff
@@ -0,0 +1,30 @@
+import app from './setupElectron';
+
+jest.setTimeout(10000);
+
+describe('Basic E2E tests', () => {
+  beforeAll(() => {
+    return app.start();
+  });
+
+  afterAll(() => {
+    if (app && app.isRunning()) {
+      return app.stop();
+    }
+  });
+
+  it('Opens a window', async function () {
```

Comment:
Test names shouldn't start with a capital letter. Also we can use an arrow function here.

```suggestion
  it('opens a window', async () => {
```